apps/www/_blog/2024-10-10-database-build-live-share.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Live Share: When your database escapes your browser'
+title: 'Live Share: connect to in-browser PGlite with any Postgres client'
```

Comment:
Perhaps capitalize accordingly:

```suggestion
title: 'Live Share: Connect to In-Browser PGlite with Any Postgres Client'
```

Reply:
@w3b6x9 how is this (to be more consistent with other posts)
```suggestion
title: 'Live Share: Connect to in-browser PGlite with any Postgres client'
```

Reply:
👍